raiden/transfer/state.py

```diff
@@ -996,15 +998,22 @@ def __repr__(self):
         )
 
     def __eq__(self, other):
+        if not isinstance(other, TransactionChannelNewBalance):
+            raise NotImplemented
```

Comment:
why was this changed? if the `other` is not a `TransactionChannelNewBalance` then the values are not equal

Reply:
apparently it should be `return` instead of `raise`. Will fix that.